### ISSUE TITLE
[BUG] IFRAME Tab ID not set correctly

### ISSUE THREAD

**Issue by dipterix** [ColorlibHQ/AdminLTE#4033]
**Describe the bug**
If sidebar item with `href` has escaped URL, then `uniqueName` name will be incorrect.

`const uniqueName = link.replace('./', '').replace(/["&'./:=?[\]]/gi, '-').replace(/(--)/gi, '')`

**To Reproduce**

Add an item in sidebar with
```
<li class="nav-item">
  <a href="/?module=widgets&shared_id=QjlIPW5cmje9xTjaCv7QoMglH8" class="nav-link">
    <i class="fa fa-th nav-icon fas" role="presentation" aria-label="th icon"></i>
    <p>
      Widgets
    </p>
  </a>
</li>
```

Search the sidebar with `widgets` and enter, the browser throws this error

```
Uncaught Error: Syntax error, unrecognized expression: #tab--%3Fmodule%3Dwidgets%26shared_id%3DQjlIPW5cmje9xTjaCv7QoMglH8
    jQuery 7
    openTabSidebar IFrame.js:148
    _setupListeners IFrame.js:257
    jQuery 2
```

**Screenshots**

**Environment (please complete the following information):**
- AdminLTE Version: [v3]
- Operating System:  [macOS Catalina]
- Browser (Version): [e.g. Firefox (Latest)]

**Additional context**

My trouble shooting has the following findings:

The search widget escapes the `href` attribute. In my case, when I searched `widgets`, the dropdown item in the search bar shows 

```
<a href="/%3Fmodule%3Dwidgets%26shared_id%3D1l8ea7phQvzbZGG9SzxaXrHB51" class="list-group-item"><div class="search-title"><strong class="text-light">Wid</strong>gets</div><div class="search-path"></div></a>
```

This `href` was used to determine the unique name via `link.replace('./', '').replace(/["&'./:=?[\]]/gi, '-').replace(/(--)/gi, '')`. 

You can see that `%` cannot be replaced, hence the created tab has the unique name containing `%`, and rejected by the browser.

The escape seems to be a good idea as the url might contain some unicodes. My poposal is that in the replacement clause, replace all characters that are not digits, letters, `_`, or `-`.

Alternatively, might just consider unescape `link` before replacing?

**Comment by dipterix** [comment]
A working modification:


https://github.com/ColorlibHQ/AdminLTE/blob/3b46ba874f623f63b01171c8632f5886f2630b78/build/js/IFrame.js#L146

changed to:

```
link = unescape(link);
var uniqueName = link.replace('./', '').replace(/["&'./:=?[\]%]/gi, '-').replace(/(--)/gi, '');
```

**Comment by REJack** [comment]
fixed with 2bbb0f1
